File: src/Modules/General/Money.cs
using Discord;
using Discord.Commands;
using System.Threading.Tasks;
using DEA.Common.Extensions;

namespace DEA.Modules.General
{
    public partial class General
    {
        [Command("Money")]
        [Alias("Cash", "Balance", "Bal")]
        [Remarks("Sexy John#0007")]
        [Summary("View the wealth of anyone.")]
        public async Task Money([Remainder] IGuildUser user = null)
        {
            user = user ?? Context.GUser;
            var dbUser = user.Id == Context.User.Id ? Context.DbUser : await _userRepo.GetUserAsync(user);

            await SendAsync($"{user.Boldify()}'s balance: {dbUser.Cash.USD()}.");
        }
    }
}

File: src/commands/general/Money.js
const db = require('../../database');
const patron = require('patron.js');
const util = require('../../utility');

class Money extends patron.Command {
  constructor() {
    super({
      name: 'bal',
      aliases: ['cash', 'balance', 'money'],
      group: 'general',
      description: 'View the wealth of anyone.',
      args: [
        new patron.Argument({
          name: 'member',
          key: 'member',
          type: 'member',
          default: patron.Default.Member,
          example:'Nibba You Cray#3333',
          remainder: true
        })
      ]
    });
  }

  async run(msg, args) {
    const dbUser = msg.author.id === args.member.id ? msg.dbUser : await db.userRepo.getUser(args.member.id, msg.guild.id);

    return util.Messenger.send(msg.channel, util.StringUtil.boldify(args.member.user.tag) + '\'s balance: ' + util.NumberUtil.format(dbUser.cash));
  }
}

module.exports = new Money();
